src/main/java/net/minestom/server/permission/DefaultPermissionHandler.java

```diff
@@ -0,0 +1,13 @@
+package net.minestom.server.permission;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A default permission handler. Used when no custom permission handler is set.
+ */
+public final class DefaultPermissionHandler implements PermissionHandler {
```

Comment:
This class should implement the changes suggested in PermissionHandler. The implementations should have the same behaviour as before.